infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -1,13 +1,14 @@
 # ===================
 # Quantum Serverless configs
 # ===================
+keycloakClientSecret: CLIENTSECRET-CHANGEME
```

Comment:
Correct me if I'm wrong @akihikokuroda but this value can be removed, can't be? I couldn't find where we are using it.

Reply:
It is actually used in the templates/rayingress.yaml.  It put the client secret into the keycloak client configuration.

Reply:
Oh, I didn't see it! Thank you @akihikokuroda . Can we moved just below `keycloakEnabled` or near to it then? To have it with all the keycloak values. Thanks! 🙏